### PR TITLE
Decouple message handling from express

### PR DESCRIPTION
**Pull request by mraaroncruz**
I want to be able to create multiple bots on different pages and forward to the correct bot based on the page id from the incoming message.
Currently this is impossible because the message handling is coupled with the express request handling.
This opens that up and also makes testing easier without needing to create an express instance to test message handling.
The tests are ugly, but I think useful.

### CODE REVIEW

**Comment by pferdefleisch** [comment]
If this is interesting, I think the next step would be to make it possible to just turn off express.

**Comment by mraaroncruz** [comment]
Regarding my last comment: It turns out you don't have to turn off express. Just don't run `bot.start()` and the express server will never be started.

**Comment by Charca** [comment]
I like this. Thanks for adding those tests, we definitely need better test coverage, and this change makes the task a lot easier.

My only request is, could you add some documentation to the readme file? I'm thinking another subsection at the end of the **Documentation** section (before **Examples**) with a description of how to use this `handleFacebookData` method and, like you said in your last comment, what should be done if you don't want to use Express.

Thanks!

File: README.md [hunk 1]
```diff
@@ -692,6 +698,39 @@ If `disableInput` is set to `true`, it will disable user input in the menu. The
...
+#### `.handleFacebookMessage(data)`
+
+Use this to send a message from a parsed webhook message directly to your bot.
+
+```js
+const linuxNewsBot   = new BootBot({argz});
+const appleNewsBot   = new BootBot({argz});
+const windowsNewsBot = new BootBot({argz});
+
+myNonExpressRouter.get("/mywebhook", (data) => {
```
Comment (felippenardi) [inline README.md]: Shouldn't this be `post`?

**Pull request merged**